.gitignore

```diff
@@ -18,3 +18,4 @@
 
 .DS_Store
 .env
+config/database.yml
```

Comment:
my linux pg requires me to set a username or else I get the "fe_auth: no password supplied" message